problemsets-mat223/linearalbegra.tex

```diff
@@ -2163,14 +2163,45 @@ \section*{Projections}
 
 	\question
 	Let $\vec a=\mat{1\\0}$, $\vec b=\mat{3\\0}$, $\vec v=\mat{1\\1}$ and
-	$\ell=\Span\{\vec a\}$.
+	$\ell=\Span\Set{\vec a}$.
 	\begin{parts}
 		\item Draw $\vec a$, $\vec b$, and $\vec v$ in the same picture.
-		\item Find $\Proj_{\{\vec b\}}\vec v$, $\Proj_{\{\vec a,\vec b\}}\vec v$.
-		\item Find $\Proj_\ell \vec v$. (Recall that a quadratic $at^2+bt+c$ has a
-			minimum at $t=-\tfrac{b}{2a}$).
+			\begin{solution}
+				XXXX
+			\end{solution}
+		\item Find $\Proj_{\Set{\vec b}}\vec v$, $\Proj_{\Set{\vec a,\vec b}}\vec v$.
+			\begin{solution}
+				$\Proj_{\Set{\vec b}}\vec v = \vec b$, since there is only one
+				point in $\Set{\vec b}$. 
+
+				$\Proj_{\Set{\vec a,\vec b}}\vec v=\vec a$. We can simply compute
+				$\norm{\vec v-\vec a} = 1$ and $\norm{\vec v-\vec b} = \sqrt{5}$,
+				so $\vec a$ is closer to $\vec v$. 
+			\end{solution}
+		\item Find $\Proj_\ell \vec v$. (Recall that a quadratic $at^2+bt+c$ has 
+			a minimum at $t=-\tfrac{b}{2a}$).
+			\begin{solution}
+				$\Proj_\ell \vec v = \vec a$.
```

Comment:
Oops! This is too big of a coincidence for students. Can you find some new numbers for the problem so that it is still really easy but we don't have something like `proj_span(a) v = a` (which students might interpret as a rule...)?

Reply:
A relatively simple change there. I could change things more to get less clean numbers if you want.